core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -777,7 +777,7 @@ private void sendToAll(ReplicableCommand rpcCommand, DeliverOrder deliverOrder)
          AnycastAddress anycastAddress = new AnycastAddress();
          dispatcher.sendMessage(anycastAddress, buffer, options);
       } else {
-         dispatcher.castMessage(null, buffer, options);
+         dispatcher.castMessage(null, buffer, options.anycasting(false));
```

Comment:
I know. I just want to be sure in case if one of us in the future change the default for some reason.